SWLOR.Game.Server/Conversation/BaseManagementTool.cs

```diff
@@ -690,6 +690,10 @@ private void DoRetrieveStructure()
                     DataService.SubmitDataChange(item, DatabaseActionType.Delete);
                     impoundedCount++;
                 }
+
+                // Reset the purchase time of the base to avoid it being confiscated.
+                pcBase.DateInitialPurchase = DateTime.UtcNow;
+                DataService.SubmitDataChange(pcBase, DatabaseActionType.Update);
```

Comment:
The only hesitation I have with changing the initial purchase date is that we're actually losing that information by changing it. It's no longer when the player actually bought the land, but when did the timer last update.

I would prefer to have us track the timer in a new column but I'm not married to the idea.